Gemfile

```diff
@@ -18,3 +18,4 @@ gem "sshkey", "1.7.0"
 gem "parallel_tests"
 gem "parallel"
 gem "librarian-puppet"
+gem 'pry'
```

Comment:
Is this needed for anything else in the PR or are we sneaking it in? I'm not against including `pry` but we should do it in a separate PR if we're adding it.